release.sh

```diff
@@ -27,126 +27,29 @@ check_git_branch() {
 
 check_git_branch
 
-# update to the latest
-git pull origin master
+sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" src/0_config.js
+sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" test/web-config.js
 
-read -p "Update CHANGELOG.md?" -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-  vi CHANGELOG.md
-  git commit -am "Updated CHANGELOG.md"
```

Comment:
we said we want to have the changelog be manual, so removed

Reply:
isn't this manual?

Reply:
i mean manual as in going into the file and making the edits yourself, then committing